bindings/ergo-lib-wasm/src/context_extension.rs

```diff
@@ -16,6 +16,18 @@ pub struct ContextExtension(
 
 #[wasm_bindgen]
 impl ContextExtension {
+    /// Create new ContextExtension instance
+    #[wasm_bindgen(constructor)]
+    pub fn new() -> Self {
+        Self(ergo_lib::ergotree_interpreter::sigma_protocol::prover::ContextExtension::empty())
+    }
+
+    /// Insert the supplied pair into the ContextExtension
+    pub fn with_pair(&mut self, id: u8, value: &Constant) -> ContextExtension {
```

Comment:
Please see my comment for `UnsignedTransaction::with_context_ext`. In this case, the method is infallible so mutating self should work fine. However, `with_*` naming is used for consuming self methods. Let's make it `pub fn set_pair(&mut self, id: u8, value: &Constant)` without return value at all.